tests/align/test_all_operators.sh

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env bash
+eval "$(conda shell.bash hook)";
+
+rm -rf align/out
+
+#create flexflow tensors
+conda activate flexflow;
+./python/flexflow_python tests/align/align_create_tensor_ff.py -ll:py 1 -ll:gpu 1 -ll:fsize 5000 -ll:zsize 4096 -b 16 -o add;
```

Comment:
Maybe make 
```
./python/flexflow_python tests/align/align_create_tensor_ff.py -ll:py 1 -ll:gpu 1 -ll:fsize 5000 -ll:zsize 4096 -b 16 -o
```
into a variable or bash function (same below with the pytorch code)

Reply:
thanks, fixed